client/layout/global-sidebar/menu-items/menu-item.jsx

```diff
@@ -0,0 +1,73 @@
+import { Gridicon } from '@automattic/components';
+import classNames from 'classnames';
+import React, { useRef, forwardRef, Fragment } from 'react';
+
+const SidebarMenuItem = forwardRef(
+	(
+		{
+			url,
+			tipTarget,
+			onClick,
+			tooltip,
+			icon,
+			className,
+			isActive,
+			preloadSection,
+			hasUnseen,
+			children,
+			alwaysShowContent,
+			disabled,
+		},
+		ref
+	) => {
+		const preloadedRef = useRef( false );
+
+		const preload = () => {
+			if ( ! preloadedRef.current && preloadSection ) {
+				preloadedRef.current = true;
+				preloadSection();
+			}
+		};
+
+		const renderChildren = () => {
+			return (
+				<Fragment>
+					{ hasUnseen && (
+						<span className="sidebar__item-bubble" aria-label="You have unseen content" />
```

Comment:
Does this actually display anything? What would be unseen about domains or sites?